lib/fluent/plugin/out_file.rb

```diff
@@ -23,9 +23,13 @@ class FileOutput < TimeSlicedOutput
       'gzip' => :gz,
     }
 
+    desc "Specify filepath to output."
     config_param :path, :string
+    desc "Choose output format. You can choose it from registered formatter plugins."
     config_param :format, :string, :default => 'out_file'
+    desc "Disable path increment."
     config_param :append, :bool, :default => false
+    desc "Compress chunks with specified compression algorithm."
```

Comment:
Ah, let me write more descriptive. 

Currently, `out_file` supports only gzip, so `Compresses flushed files using gzip` is not wrong, but I think 

    Compress flushed file with specified compression algorithm.

is fine. In addition, documents should list what algorithms can be specified.